ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Image/ImagePageVulnerabilities.tsx

```diff
@@ -327,16 +228,137 @@ function ImagePageVulnerabilities({
             >
                 <VulnerabilityStateTabs isBox onChange={() => setPage(1)} />
                 <div className="pf-v5-u-px-sm pf-v5-u-background-color-100">
-                    <WorkloadCveFilterToolbar
-                        searchOptions={searchOptions}
-                        autocompleteSearchContext={{
-                            'Image SHA': imageId,
-                        }}
-                        onFilterChange={() => setPage(1)}
-                    />
+                    {isAdvancedFiltersEnabled ? (
+                        <AdvancedFiltersToolbar
+                            className="pf-v5-u-pt-lg pf-v5-u-pb-0"
+                            searchFilterConfig={searchFilterConfig}
+                            searchFilter={searchFilter}
+                            onFilterChange={(newFilter, { action }) => {
+                                setSearchFilter(newFilter);
+                                setPage(1, 'replace');
+
+                                if (action === 'ADD') {
+                                    // TODO - Add analytics tracking ROX-24532
+                                }
+                            }}
+                        />
+                    ) : (
+                        <WorkloadCveFilterToolbar
+                            searchOptions={searchOptions}
+                            autocompleteSearchContext={{
+                                'Image SHA': imageId,
+                            }}
+                            onFilterChange={() => setPage(1)}
+                        />
+                    )}
```

Comment:
Since inlining the `mainContent` variable here as a refactor, this highlighted section is effectively the only changed section of code within the JSX.